src/lib/hooks/useBidData.ts

```diff
@@ -6,8 +6,8 @@ export const useBidData = (domainId: string) => {
 	const sdk = useZnsSdk();
 
 	return useQuery(
-		['domain-bids', domainId],
-		async () => await sdk.zauction.listBids(domainId),
+		['domain', 'bids', { domainId }],
+		() => sdk.zauction.listBids(domainId),
```

Comment:
Oh async and await not required here cool, good to know!

Reply:
Yep! `sdk.zauction.listBids` returns a promise which `react-query` awaits internally.